### PR TITLE
Use proper display name for workspace roles

### PR DESCRIPTION
**Pull request by richard-dds**
The members list screen was showing roles using `Role.name`, which presented weird-looking strings like `billing_auditor` and `ccpo`. This PR adds a `Role.display_name` column, seeds it from `seed_roles.py`, and displays it in `members/index.html`.

screen shot 2018-09-24 at 2 22 18 pm

### CODE REVIEW

File: alembic/versions/c19ae79d2521_add_role_display_name.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# revision identifiers, used by Alembic.
+revision = "c19ae79d2521"
+down_revision = "359caaf8c5f1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column(
```
Comment (patricksmithdds) [inline alembic/versions/c19ae79d2521_add_role_display_name.py]: Is this change intended?

File: script/seed_roles.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 roles = [
     Role(
         name="ccpo",
+        display_name="CCPO",
```
Comment (patricksmithdds) [inline script/seed_roles.py]: There's a couple descriptions available in `forms/data.py` for some of these roles as well. Could we add the descriptions here, and then instead of hard-coding stuff in `forms/data.py`, just look up the subset of available roles where we need them?

File: alembic/versions/c19ae79d2521_add_role_display_name.py (hunk 1 above)
Comment (richard-dds) [reply alembic/versions/c19ae79d2521_add_role_display_name.py]: It wasn't, but alembic added it with `--autogenerate`, and it does seem necessary. I thought about opening up another PR to add it, but it seemed like more trouble than it was worth.

File: script/seed_roles.py (hunk 2 above)
Comment (richard-dds) [reply script/seed_roles.py]: We could, but I'm not exactly sure what the right way would be to do that with WTForms. Could we punt on it? Currently we're just not using `Role.description`.

File: alembic/versions/c19ae79d2521_add_role_display_name.py (hunk 1 above)
Comment (patricksmithdds) [reply alembic/versions/c19ae79d2521_add_role_display_name.py]: Makes sense to me.

File: script/seed_roles.py (hunk 2 above)
Comment (patricksmithdds) [reply script/seed_roles.py]: That makes sense. I've put a `tech debt` issue in pivotal for reconciling the possible duplication.
